pkg/cmd/cli/backup/describe.go

```diff
@@ -85,6 +86,7 @@ func NewDescribeCommand(f client.Factory, use string) *cobra.Command {
 
 	c.Flags().StringVarP(&listOptions.LabelSelector, "selector", "l", listOptions.LabelSelector, "only show items matching this label selector")
 	c.Flags().BoolVar(&details, "details", details, "display additional detail in the command output")
+	c.Flags().BoolVar(&insecureSkipVerify, "insecureskipverify", insecureSkipVerify, "do not verify the TLS certificate for storage requests. This is susceptible to man-in-the-middle attacks.")
```

Comment:
Any reason to not have this be `insecure-skip-verify`?  It might also be a good idea to use the same flag `kubectl` uses: `insecure-skip-tls-verify`.

Reply:
Good point 👍

Reply:
good idea. changed it all to `insecure-skip-tls-verify`